.github/ISSUE_TEMPLATE.md

```diff
@@ -17,7 +17,7 @@ If the node is "stuck" during sync or giving "block checksum mismatch" errors, p
 
 <!-- What type of machine are you observing the error on (OS/CPU and disk type)? -->
 
-<!-- For the GUI-related issue on Linux provide names and versions of a distro, a desktop environment and a graphical shell (if relevant). -->
+<!-- GUI-related issue? What is your operation system and its version? If Linux, what is your desktop environment and graphical shell? -->
 
 <!-- Any extra information that might be useful in the debugging process. -->
 <!--- This is normally the contents of a `debug.log` or `config.log` file. Raw text or a link to a pastebin type site are preferred. -->
```

Comment:
These `<!--- ` could be explained as a leading MD `- `, but don't look great in the issue preview plaintext and never get rendered as Markdown. Maybe instead add a `:` e.g. `Describe the issue:`?

Reply:
I think it is fine to keep this as it was previously